lib/power_action_utils.pm

```diff
@@ -174,9 +174,11 @@ sub poweroff_x11 {
     }
 
     if (check_var('BACKEND', 's390x')) {
+        diag('Skipping check for the SUT to be shut down using x3270, due to poo#');
+        return;
         # make sure SUT shut down correctly
         console('x3270')->expect_3270(
-            output_delim => qr/.*SIGP stop.*/,
+            output_delim => qr/SIGP stop/,
```

Comment:
Do we need this change, as we return earlier?

Reply:
I just tried several options to see if it will work. I'll revert it, as it did not help.